articles/virtual-machines/workloads/oracle/configure-oracle-dataguard.md

```diff
@@ -587,27 +659,27 @@ cdb1_stby=
 Start SQL*Plus:
 
 ```bash
-sqlplus sys/OraPasswd1@cdb1
-SQL*Plus: Release 12.2.0.1.0 Production on Wed May 10 14:18:31 2017
+$ sqlplus sys/OraPasswd1@cdb1
+SQL*Plus: Release 19.0.0.0 Production on Wed May 10 14:18:31 2022
 
 Copyright (c) 1982, 2016, Oracle.  All rights reserved.
 
 Connected to:
-Oracle Database 12c Enterprise Edition Release 12.1.0.2.0 - 64bit Production
+Oracle Database 12c Enterprise Edition Release 19.0.0.0 - 64bit Production
 With the Partitioning, OLAP, Advanced Analytics and Real Application Testing options
 
 SQL>
 ```
 
 ## Test the Data Guard configuration
```

Comment:
Hi @glbwell0 For H2s we usually add an introductive paragraph (a few sentences) describing what the customer will learn about or do throughout that section, like a high-level summary of all the H3s & describing what to expect. Can you please add one under the H2.